src/browser.js

```diff
@@ -40,7 +40,7 @@ function useColors() {
   // NB: In an Electron preload script, document will be defined but not fully
   // initialized. Since we know we're in Chrome, we'll just detect this case
   // explicitly
-  if (window && window.process && window.process.type === 'renderer') {
+  if (window && window.process && window.process.type === 'renderer' || !window.document) {
```

Comment:
I think that the point wasn't just to detect react-native, but to detect a lack of `window.document` which is used in the next part of the check.